tutorials/groundwater-recharge-estimation/index.ipynb

```diff
@@ -1027,8 +1027,8 @@
       },
       "outputs": [],
       "source": [
-        "# Combine precipitation and evapotranspiration.\n",
-        "meteo = pr_m.combine(pet_m)\n",
+        "# Combine precipitation and evapotranspiration, and sort the collection.\n",
```

Comment:
No need to mention the sort in the comment. That's redundant to what's in the code.